wp-seo-main.php

```diff
@@ -11,7 +11,7 @@
 }
 
 /**
- * @internal Nobody should be able to overrule the real version number as this can cause serious issues
+ * @note Nobody should be able to overrule the real version number as this can cause serious issues
```

Comment:
Well, @internal means nobody outside the project or module should call this function.

You can disable this inspection which was added in 8.0 (https://youtrack.jetbrains.com/issue/WI-22284)